data.json

```
@@ -766,5 +766,11 @@
     "githubUsername": "Rishabhrathore055",
     "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
     "issueNumber": 445
-  }
+  },
+  {
+    "name": "Advaith U",
+    "githubUsername": "advaith-unnikrishnan",
+    "imageUrl": "https://user-images.githubusercontent.com/45172876/115498803-d73a2d00-a28b-11eb-809c-79a8eaf10a5d.png",
+    "issueNumber": 447
+}
```

please line up with the other braces
```suggestion
  }
```